src/analysis/utils/tool-utils.ts

```diff
@@ -20,11 +20,11 @@ export const launchableToolLabel: Record<LaunchableToolLabel, LaunchableToolLabe
   spark: 'spark',
 };
 
-export const runtimeToolLabels: Record<RuntimeToolLabel, RuntimeToolLabel> = {
+export const runtimeToolLabels = {
   Jupyter: 'Jupyter',
   RStudio: 'RStudio',
   JupyterLab: 'JupyterLab',
-};
+} as const satisfies Record<RuntimeToolLabel, RuntimeToolLabel>;
```

Comment:
The `const` assertion improves the types for "enums" like this. Now the type of `runtimeToolLabels.Jupyter` is "Jupyter" instead of `RuntimeToolLabel`. This lets us use this enum when creating objects that satisfy `Record<RuntimeToolLabel, x>`.

https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes-func.html#readonly-and-const